kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -96,6 +99,11 @@ def __init__(
         self.log_file_path = setup_logger(log_dir, **log_kwargs)
         self.logger = logging.getLogger(type(self).__name__)
 
+        head, tail = os.path.split(self.log_file_path)
+        self.counts_file_path = os.path.join(head, 'counts_for_' + tail)
+        with contextlib.suppress(FileNotFoundError):
```

Comment:
> Why not use try/except?

🤷‍♂ 2 lines (import+suppress) vs 3 (try+except+pass)? tomato tomato

> shouldn't we clear all previous stage output rather than just the count log..why did we decide to do it for just count logs now

The counts file was originally written piecemeal in append mode after each stage. That's no longer done, so I'll just remove this.

Reply:
> All of the files naturally replace themselves

Only if the current run is overwriting existing files. What if someone removed an extract config from their package and then re-ran? The extract output for the removed extract config would still be there bc there is nothing clearing out the entire output folder. I think we should be doing this so that each run has a fresh set of output

Reply:
Probably should do that ^ in another PR tho